textsplitter/options.go

```diff
@@ -0,0 +1,32 @@
+package textsplitter
+
+// Option is a function that configures Options.
+type Option func(*Options)
+
+// Options is a set of options for configuring a text splitter.
+type Options struct {
```

Comment:
Please let me know if this needs to be an option that's specific to `RecursiveCharacter` and not a package-wide option.

Reply:
I'm also happy to un-export the fields in `RecursiveCharacter` if needed, given they'll be configured by the options.